packages/rspack-test-tools/tests/configCases/web/fetch-priority-2/index.js

```diff
@@ -1,42 +1,42 @@
 it("should set fetchPriority", () => {
 	import(/* webpackFetchPriority: "high" */ "./a");
-	expect(document.head.children).toHaveLength(3);
-	const script1 = document.head.children[1];
+	expect(document.head.children).toHaveLength(4);
```

Comment:
Because now we can detect prefetch correctly, negative prefetch is ignored before